openpifpaf/network/losses/composite.py

```diff
@@ -104,13 +112,16 @@ def _localization_loss(self, x_regs, t_regs):
             )
             if self.prescale != 1.0:
                 loss = loss * self.prescale
+            weight = torch.ones_like(reg_masks, dtype=torch.float, requires_grad=False)
```

Comment:
Here also, remove the explicit dtype. However, the mask will have the wrong dtype. You can use `t_regs[:, :, 0]` instead.